src/wiki/plugins/pymdown/settings.py

```diff
@@ -27,3 +31,78 @@ def update_whitelist():
     settings.MARKDOWN_KWARGS["extension_configs"].update(
         getattr(django_settings, "PYMDOWNX_KEYWORD_ARGUMENTS", dict())
     )
+
+
+pymdown_docs = [
+    {
+        "short_description": _("Admonition"),
+        "help_text": _("Insert an Admonition html tag"),
+        "example_code": """
+/// admonition | Some title
+    type: warning
+
+Some content
+///
+More information: https://facelessuser.github.io/pymdown-extensions/extensions/blocks/plugins/admonition/
```

Comment:
I found it confusing that this text is included in the example. At first, I thought that it was part of the syntax, then realized that it's a helpful note.

Can it be moved to the `help_text` part? (same for the other ones)

Reply:
Done. I added it to the help_text part. Thanks for catching that. Doesn't belong in the example code section.

Reply:
Great! Thanks! :)